skynet/modules/ttt/summaries/processor.py

```diff
@@ -33,9 +33,11 @@ def initialize():
     global llm
 
     llm = ChatOpenAI(
+        model=llama_path,
```

Comment:
Do we need the full path? Or `os.path.basename(llama_path)` works?

Reply:
what would that return when in Dockerfile we specify it as `LLAMA_PATH="/models/Llama-3.1-8B-Instruct-Q8_0.gguf"`?

Reply:
`Llama-3.1-8B-Instruct-Q8_0.gguf` basename is the last part of a path.

Reply:
Does it need the full path?

Reply:
yup

Reply:
The disregard my comment :-P